tests/osaka/eip7692_eof_v1/eip4200_relative_jumps/test_rjumpi.py

```diff
@@ -1002,3 +1644,183 @@ def test_double_rjumpi_invalid_max_stack_height(
+            sections=[
+                Section.Code(
+                    code=Op.PUSH0
+                    + Op.POP
+                    + Op.PUSH1[0]
+                    + Op.RJUMPI[-7]
+                    + Op.PUSH0
+                    + Op.RJUMP[-11],
+                    max_stack_height=1,
+                ),
+            ],
+            expected_bytecode="ef0001010004020001000b04000000008000015f506000e1fff95fe0fff5",
+        ),
+        Container(
+            name="backwards_rjumpi_9",
+            sections=[
+                Section.Code(
+                    code=Op.PUSH0
+                    + Op.PUSH1[0]
+                    + Op.RJUMPI[1]
+                    + Op.PUSH0
+                    + Op.PUSH1[0]
+                    + Op.RJUMPI[-11]
+                    + Op.STOP,
+                    max_stack_height=3,
+                ),
+            ],
+            expected_bytecode="ef0001010004020001000d04000000008000035f6000e100015f6000e1fff500",
+        ),
+        Container(
+            name="backwards_rjumpi_variable_stack_3",
+            sections=[
+                Section.Code(
+                    code=Op.PUSH0
+                    + Op.PUSH1[0]
+                    + Op.RJUMPI[2]
+                    + Op.PUSH0
+                    + Op.PUSH0
+                    + Op.PUSH0
+                    + Op.POP
+                    + Op.PUSH1[0]
+                    + Op.RJUMPI[-7]
+                    + Op.PUSH0
+                    + Op.PUSH1[0]
+                    + Op.RJUMPI[-13]
+                    + Op.STOP,
+                    max_stack_height=5,
+                ),
+            ],
+            expected_bytecode="ef0001010004020001001604000000008000055f6000e100025f5f5f506000e1fff95f6000e1fff300",
+        ),
+        Container(
+            name="backwards_rjumpi_variable_stack_5",
+            sections=[
+                Section.Code(
+                    code=Op.PUSH0
+                    + Op.PUSH1[0]
+                    + Op.RJUMPI[2]
+                    + Op.PUSH0
+                    + Op.PUSH0
+                    + Op.PUSH0
+                    + Op.PUSH1[1]
+                    + Op.ADD
+                    + Op.DUP1
+                    + Op.DUP1
+                    + Op.RJUMPI[-8]
+                    + Op.STOP,
+                    max_stack_height=5,
+                ),
+            ],
+            expected_bytecode="ef0001010004020001001204000000008000055f6000e100025f5f5f6001018080e1fff800",
+        ),
+        Container(
+            name="backwards_rjumpi_variable_stack_6",
+            sections=[
+                Section.Code(
+                    code=Op.PUSH0
+                    + Op.PUSH1[0]
+                    + Op.RJUMPI[2]
+                    + Op.PUSH0
+                    + Op.PUSH0
+                    + Op.PUSH0
+                    + Op.PUSH0
+                    + Op.PUSH0
+                    + Op.POP
+                    + Op.RJUMPI[-4]
+                    + Op.STOP,
```

Comment:
```suggestion
                    code=Op.PUSH0
                    + Op.PUSH1[0]
                    + Op.RJUMPI[2]
                    + Op.PUSH0
                    + Op.PUSH0
                    + Op.PUSH0
                    + Op.PUSH0
                    + Op.POP
                    + Op.PUSH0
                    + Op.RJUMPI[-5]
                    + Op.STOP,
```
Slight variation to this case that could be interesting?

Reply:
Added new variant.